### ISSUE TITLE
Replace expiration date picker with native date picker

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#33914]
Replace old third library date picker with native one https://github.com/nextcloud/nextcloud-vue/issues/1666

![image](https://user-images.githubusercontent.com/6078378/188614726-c1b5f003-c2d0-4b3b-ba7c-58d5ee133834.png)

Todo: moment.js dates should be replaced. Usage of moment.js makes no sense for native date picker anymore.

**Comment by JuliaKirschenheuter** [comment]
PR: https://github.com/nextcloud/server/pull/33915
